web/src/invite.js

```diff
@@ -256,6 +257,9 @@ function open_invite_user_modal(e) {
 
     function invite_user_modal_post_render() {
         $("#invite-user-modal .dialog_submit_button").prop("disabled", true);
+        for (const stream of streams.filter((stream) => stream.default_stream)) {
+            $(`input[value=${stream.stream_id}]`).prop("checked", true);
+        }
```

Comment:
This should be moved to the commit which changes the template code to not handle marking the default streams initially and the reason for this change should be mentioned in the commit message